Retrie/ExactPrint.hs

```diff
@@ -4,6 +4,7 @@
 -- LICENSE file in the root directory of this source tree.
 --
 {-# LANGUAGE CPP #-}
+{-# LANGUAGE FlexibleContexts #-}
```

Comment:
I don't see this used... added during debugging?

Reply:
yes, gone now